java/java.source.base/src/org/netbeans/modules/java/source/save/Reformatter.java

```diff
@@ -2809,6 +2815,41 @@ private Boolean scanBindingPattern(Tree node, Void p) {
             return true;
         }
 
+        private Boolean scanGuardedPattern(Tree node, Void p) {
+            try {
+                Tree gpt = TreeShims.getGuardedPattern(node);
+                if (gpt != null) {
+                    Name name = TreeShims.getBinding(gpt);
+                    Tree type = TreeShims.getBindingPatternType(gpt);
+                    scan(type, p);
+                    if (name != null) {
+                        removeWhiteSpace(IDENTIFIER);
```

Comment:
It is not quite clear what `remoteWhitespace` does - would `space();` work?

Reply:
space(); also worked here, but adding some extra black spaces after case labels. In some case wrapping extra blank line also doesn't worked. If you want, I'm fine to replace this with space();